src/content/docs/get-started/release.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Release information
+description: Determine which versions of the drop-in components are compatible with each other and with Adobe Commerce.
+---
+
+Adobe organizes storefront application releases into logical suite versions and follow a regular release schedule (approximately once a month). However, Adobe may release updates as necessary.
+
+## Compatibility table
+
+The following table shows the compatibility between all of the storefront release items by suite version:
+
+| Release item | Version | |
+|---|---|---|
+| **Storefront suite** | 1.0.0 |
+| **Released** | December 2024 |
+| **Adobe Commerce** | 2.4.7 |
+| **Storefront compatibility package** | 4.7.0 |
+| **Drop-in SDK** | 0.37.0 |
+| **Drop-in components** | Product Details: 1.0.0<br/>Cart: 1.0.0<br/>Checkout: 1.0.0<br/>User Auth: 1.0.0<br/>User Account: 1.0.0<br/>Orders: 1.0.0 |
+
+## Code changes
+
+Every release includes changes to one or more codebases listed above. The scope of the change determines whether the MAJOR, MINOR, or PATCH number increases in the application's version according to [semantic versioning](https://semver.org/):
+
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+> 
+> MAJOR version when you make incompatible API changes  
+> MINOR version when you add functionality in a backward compatible manner  
+> PATCH version when you make backward compatible bug fixes  
+
+Note: The "Storefront suite" version is more of a marketing construct and does not conform to semantic versioning.
+
+### Public API
+
+In order to determine when we increment the MAJOR version we must define a Public API.
+
+The Public API for drop-ins includes the following:
+
+#### Integration APIs
+
+1. Function signatures of public functions in Drop-in SDK tools
+  * `fetch-graphql`
+    * `setEndpoint`
+    * `setFetchGraphQlHeader`
+    * `removeFetchGraphQlHeader`
+    * `setFetchGraphQlHeaders`
+    * `fetchGraphQl`
+    * `getConfig`
+  * `event-bus`?
+  * [Elsie exports](https://git.corp.adobe.com/Commerce-Storefront/StorefrontSDK/blob/82df5ab3b9f23314c2b0816628c928b43643f74b/packages/elsie/src/lib/index.ts)?
+    * Function signatures of `render` and `initialize`
+
+#### Slots
+
+1. Default properties available to all slots:
+  * Function signatures of functions `prependSibling`, `prependChild`, `replaceWith`, `appendChild`, `appendSibling`, `getSlotElement`, `onChange`
+  * Structure of `dictionary` object
+2. Function signatures of individual drop-in slots as defined elsewhere on this site
+  * [Cart slots](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/cart/slots/), for example
+
+#### Drop-in design tokens
+
+1. Drop-in design tokens defined by the [drop-in SDK](https://elsie.dev/?path=/docs/design-overview--overview)
```

Comment:
We probably don't want to actually link to this development documentation site. Not sure how best to enumerate all of these tokens otherwise.

Reply:
We might be able to pull in a sunset of the SDK docs via component that @bdenham developed in this repo: https://github.com/commerce-docs/dropins.dev

Reply:
Actually, can we just link here @supernova-at?

https://experienceleague.adobe.com/developer/commerce/storefront/dropins/all/branding/#design-tokens

Reply:
Yes, perfect!